### PR TITLE
work on dashboard

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

A few drive-by changes, resolving most of the final issues from [alpha2 feedback](https://docs.google.com/document/d/1QeRghWCFdJgWV627DaIWRM80pJ3IkJr_TRQjI4uJXEw/edit#):

* status messaging
* consistent button labels on dashboard
* link styling fix

Work on dashboard:

* styling updates
* bar chart
* lessons

> image

### Reviewer guidance

dashboard should be fully data-driven now

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Gherkin stories have been updated
- [ ] Unit tests have been updated

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/home/HomePage/LessonsBlock.vue [hunk 1]
```diff
@@ -48,6 +47,62 @@
...
+    computed: {
+      ...mapState('classSummary', ['groupMap']),
+      ...mapGetters('classSummary', [
+        'learners',
+        'lessons',
+        'lessonLearnerStatusMap',
+        'getLessonStatusCounts',
+      ]),
+      table() {
+        const recent = sortBy(this.lessons, this.lastActivity).slice(0, MAX_LESSONS);
```
Comment (jonboiser) [inline kolibri/plugins/coach/assets/src/views/home/HomePage/LessonsBlock.vue]: The `lastActivity` method is returning the right dates, but `sortBy` isn't sorting them correctly..

Comment (jonboiser) [reply kolibri/plugins/coach/assets/src/views/home/HomePage/LessonsBlock.vue]: Use this instead

```
        const recent = orderBy(this.exams, this.lastActivity, ['desc']);

```

File: kolibri/plugins/coach/assets/src/views/home/HomePage/QuizzesBlock.vue [hunk 2]
```diff
@@ -42,8 +48,8 @@
     },
     computed: {
       ...mapState('classSummary', ['groupMap', 'examLearnerStatusMap']),
-      ...mapGetters('classSummary', ['learners', 'exams']),
-      recentQuizzes() {
+      ...mapGetters('classSummary', ['learners', 'exams', 'getExamStatusCounts']),
+      table() {
         const recent = sortBy(this.exams, this.lastActivity).slice(0, MAX_QUIZZES);
```
Comment (jonboiser) [inline kolibri/plugins/coach/assets/src/views/home/HomePage/QuizzesBlock.vue]: Use `orderBy` with 'desc' argument, like in the LessonsBlock

**Review by jonboiser — CHANGES_REQUESTED** [review]

**Comment by indirectlylit** [comment]
thanks @jonboiser! comments addressed

**Review by jonboiser — APPROVED** [review]

**Pull request merged**